### PR TITLE
STAC-8619: Document sync merge strategies

### PR DESCRIPTION
**Pull request by craffit**

### CODE REVIEW

File: configure/sync.md [hunk 1]
```diff
@@ -43,6 +43,17 @@ Next up is mapping. It specifies the transformation of external system topologic
...
 * _Merge Strategy_ - indicates the merge strategy applied in case several components form a single entity
 
+## Merge Strategy
+
+The merge strategy is applied when multiple components get synchronized which have the same 'identifier'. At this point StackState has to decide what data to put into the resulting component, this is what the merge strategy configures. One of the following 4 options can be chosen:
+
+* _Use Mine only_ - Discard the other components, use just the result of the current mapping
+* _Use Theirs always_ - Discard the result of this mapping, go with the result of the other merged components
+* _Merge, prefer mine_ - Merge fields that are mergable, if not take the data from this mapping
+* _Merge, prefer theirs_ - Merge fields that are mergable, if not take the dat from the other components
```
Comment (whazenberg) [inline configure/sync.md:53]: dat = data

File: configure/sync.md [hunk 2]
```diff
@@ -43,6 +43,17 @@ Next up is mapping. It specifies the transformation of external system topologic
...
+
+Mergable fields in the component are set fields (like streams and checks) and optional fields (like version, description).
```
Comment (whazenberg) [inline configure/sync.md:55]: Suggestion to add: Other mandatory fields (like name) is not mergeable and the mine or theirs side will be used only.

File: configure/sync.md (hunk 1 above)
Comment (craffit) [reply configure/sync.md]: done

File: configure/sync.md (hunk 2 above)
Comment (craffit) [reply configure/sync.md]: done

**Pull request merged**